bin/rainforest_cd

```diff
@@ -3,18 +3,14 @@
 # Die on error
 set -e
 
-gem install rainforest-cli
-
-echo $PATH
-ruby --version
-gem list
-rvm gemset list
-
 # Run all Rainforest tests. RAINFOREST_TOKEN is an environment variable
 # that needs to be configued in CircleCI's UI. It represents the API token
 # of your Rainforest account. You can find it in your account's settings.
 rainforest run all --token $RAINFOREST_TOKEN --fg --fail-fast
```

Comment:
Since there's the `mv /usr/local/bin/rainforest /usr/local/bin/rainforest-cli`, this line needs to be updated. Or you can get rid of the mv above and leave this as just `rainforest run`(I think that's probably preferable). I'm not sure why we rename it to rainforest-cli? I guess that's kinda up to this repo to decide on what the best practice is

```suggestion
rainforest-cli run all --token $RAINFOREST_TOKEN --fg --fail-fast
```

Reply:
I just removed the renaming.